### PR TITLE
Capstone project

### PR DESCRIPTION
**Pull request by Graciano1997**
Hello! 
In this Project I :

-Added the home page.
- Added the About page.
- Add media query (Mobile: up to 768px wide Desktop: 768px or wider) .
- Added link Interactions 
- Mobile menu
- Dynamic page
- Updated Readme file
- I fix the linters errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+                    <li>My Page</li>
+                    <li>Logout</li>
+                </ul>
+            </div>
+
+            <button class="hamburguer" type="button"><img src="./assets/img/hamburguer.svg" alt="hamburguer icon"></button>
+            <div class="secondNavbar">
+                <div class="logoContainer">
+                    <a href="index.html"><img src="./assets/img/logo.png" alt="Summit logo"></a>
+                </div>
```
Comment (1GORDON) [inline index.html:33]: Great job @Graciano1997 🤝

- [ ] As stated in the project requirements, kindly endeavor to stick to the design guidelines as much as possible. You may remove the logo from both the Home and the About versions of your mobile views.
- Your design
your-mobile
- Given design
given-mobile

**Review by 1GORDON — CHANGES_REQUESTED** [review]
##
## Hi @Graciano1997 ✋

Good job so far!

## Highlights!
- Project deployed ✔️
- Responsive design ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Graciano1997) [reply index.html:33]: thanks for the recommendation

**Review by cynthiainga — APPROVED** [review]
Hi @Graciano1997, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

- [ ] Good design ✔️ 
- [ ] There are `Home` and `About` pages ✔️ 
- [ ] No linters errors ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**